packages/teleport-plugin-react-proptypes/src/index.ts

```diff
@@ -37,10 +37,16 @@ export const createPlugin: ComponentPluginFactory<ReactJSPropTypesConfig> = (con
     }
 
     // TODO used the name from the mappings of the component, not from the UIDL
-    const defaultPropsAst = buildDefaultPropsAst(name, uidl.propDefinitions)
+    let hasDefaultProps: boolean = false
+    Object.keys(uidl.propDefinitions).forEach((prop) => {
```

Comment:
you can use Array.prototype.some here to avoid doing a condition - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some